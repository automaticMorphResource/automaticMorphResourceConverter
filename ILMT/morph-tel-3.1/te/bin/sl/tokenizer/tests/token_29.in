మాదానాయుడి తండ్రి సింగమనాయుడు దుర్గం కొంత భాగం నిర్మాణం పూర్తి చేసినా, పూర్థి స్థాయిలో మాదానాయుడి కాలంలోనే రూపొందినట్లు తెలుస్తోంది . 
