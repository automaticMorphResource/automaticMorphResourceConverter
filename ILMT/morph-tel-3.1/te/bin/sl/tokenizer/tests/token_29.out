<Sentence id="1">
1	మాదానాయుడి	unk
2	తండ్రి	unk
3	సింగమనాయుడు	unk
4	దుర్గం	unk
5	కొంత	unk
6	భాగం	unk
7	నిర్మాణం	unk
8	పూర్తి	unk
9	చేసినా	unk
10	,	unk
11	పూర్థి	unk
12	స్థాయిలో	unk
13	మాదానాయుడి	unk
14	కాలంలోనే	unk
15	రూపొందినట్లు	unk
16	తెలుస్తోంది	unk
17	.	unk
</Sentence>

